src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -102,14 +103,14 @@ class IOUAmountPage extends React.Component {
     }
 
     /**
-     * Update amount with number or Backspace pressed.
+     * Update amount with number or Backspace pressed for BigNumberPad.
      * Validate new amount with decimal number regex up to 6 digits and 2 decimal digit to enable Next button
      *
      * @param {String} key
      */
-    updateAmountIfValidInput(key) {
+    updateAmountNumberPad(key) {
         // Backspace button is pressed
-        if (key === '<' || key === 'Backspace') {
+        if (key === '<') {
```

Comment:
Hmmm why this change?

Reply:
Because the `key` can never be `Backspace`.

Reply:
Hmmm but technically you can have a small screen on a device with a keyboard (i.e: make my desktop or browser narrow), and there's no reason specifically to disable this.

Reply:
i agree